package.json

```diff
@@ -21,19 +21,17 @@
     "url": "http://projects.theforeman.org/projects/foreman_discovery/issues"
   },
   "peerDependencies": {
-    "@theforeman/vendor": "^6.0.0"
+    "@theforeman/vendor": ">= 6.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^6.0.0",
-    "@theforeman/eslint-plugin-foreman": "^6.0.0",
-    "@theforeman/stories": "^7.0.0",
-    "@theforeman/test": "^8.0.0",
-    "@theforeman/vendor-dev": "^6.0.0",
-    "babel-eslint": "^10.0.3",
-    "eslint": "^6.7.2",
-    "prettier": "^1.19.1",
-    "stylelint": "^9.3.0",
-    "stylelint-config-standard": "^18.0.0"
+    "@babel/core": ">= 7.7.0",
+    "@theforeman/builder": ">= 6.0.0",
+    "@theforeman/eslint-plugin-foreman": ">= 6.0.0",
+    "@theforeman/stories": ">= 7.0.0",
+    "@theforeman/test": ">= 8.0.0",
```

Comment:
https://github.com/theforeman/foreman_host_reports/pull/24 got in, so you can modify it in the same way:
```json
"peerDependencies": {
    "@theforeman/vendor": ">= 0"
  },
  "dependencies": {
    "react-json-tree": "^0.11.0"
  },
  "devDependencies": {
    "@babel/core": "^7.7.0",
    "@theforeman/builder": ">= 0",
    "@theforeman/eslint-plugin-foreman": ">= 0",
    "@theforeman/stories": ">= 0",
    "@theforeman/test": ">= 0",
```

Reply:
Just like that? Rebased.